policy/triage-bugzilla.md

```diff
@@ -59,6 +59,15 @@ Also, as a release approaches, the release status of open, high priority (P1) bu
 
 An explanation of the release status field.
 
+### Automatic Triage Overrides
+
+When a bug is tracked for a release, i.e. the `tracking_firefoxNN` flag is set to `+` triage decisons will be overriden, or made as follows:
```

Comment:
'+' or 'blocking' for the tracking flag value
"decisions" instead of "decisons"